bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -906,22 +936,40 @@ public void stop() {
 	webView.Stop();
 }
 
+private boolean isLocationForCustomText(String location) {
+	try {
+		return URI_FOR_CUSTOM_TEXT_PAGE.equals(URI.create(location));
```

Comment:
I would rather use new URI, as URI.create is meant for situations where you are sure URI is valid.

Reply:
Indeed, thank you for the hint! @amartya4256 can you please adapt the code?

Reply:
Done